core/src/main/resources/org/apache/calcite/runtime/CalciteResource.properties

```diff
@@ -319,4 +319,5 @@ InvalidInputForExtractXml=Invalid input for EXTRACT xpath: ''{0}'', namespace: '
 InvalidInputForExistsNode=Invalid input for EXISTSNODE xpath: ''{0}'', namespace: ''{1}''
 DifferentLengthForBitwiseOperands=Different length for bitwise operands: the first: {0,number,#}, the second: {1,number,#}
 NoOperator=No operator for ''{0}'' with kind: ''{1}'', syntax: ''{2}'' during JSON deserialization
+IllegalArrayExpression=Illegal array expression {0}
```

Comment:
please put the line in the correct place in the file; putting it at the end tends to create merge conflicts

Reply:
I don't see any tests for 'Illegal array expression'. Please add at least one test.